deploy/olm-catalog/service-assurance-operator/service-assurance-operator.package.yaml

```diff
@@ -1,5 +1,5 @@
 channels:
-- currentCSV: service-assurance-operator.v0.1.0
-  name: alpha
-defaultChannel: alpha
+- currentCSV: service-assurance-operator.v0.1.1
+  name: beta
```

Comment:
:tada:

Reply:
@leifmadsen Does https://github.com/redhat-service-assurance/service-assurance-operator/pull/27/files#diff-7ec4e59368790398a0b5c9b240469927L8 to match this "beta"?

Reply:
I don't think they need to match. One is an API version, and the other is a channel version / name. It might make sense for things to match though. At some point we could probably justify dropping the `alpha` or `beta` parts directly and just calling it `v1`?